components/shipengine/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@pipedream/shipengine",
+  "version": "0.0.1",
+  "description": "Pipedream Shipengine Components",
+  "main": "shipengine.app.mjs",
+  "keywords": [
+		"pipedream",
+		"shipengine"
+  ],
+  "homepage": "https://pipedream.com/apps/shipengine",
+  "author": "Pipedream <support@pipedream.com> (https://pipedream.com/)",
+  "license": "MIT",
+  "dependencies": {
+    "@pipedream/platform": "^1.1.1"
```

Comment:
please use the newest version of the package `@pipedream/platform`, which is version 1.2.1;
https://www.npmjs.com/package/@pipedream/platform